.github/workflows/scripts/detect-all-changes.sh

```
@@ -0,0 +1,175 @@
+  
+  if git rev-parse --verify "$tag_name" >/dev/null 2>&1; then
+    echo "      ⏭️ Tag already exists"
+    continue
+  fi
+  
+  latest_tag=$(git tag -l "plugins/${plugin_name}/v*" | sort -V | tail -1)
+  if [ -z "$latest_tag" ]; then
+    echo "      ✅ First release"
+    PLUGIN_CHANGES+=("$plugin_name")
+  else
+    previous_version=${latest_tag#plugins/${plugin_name}/v}
```

Shellcheck (0.10.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)